src/listener/utils/transforms/plain-text-transformer.js

```diff
@@ -137,6 +140,10 @@ export default class FlowdockTransform {
   }
 
   getProviderName() {
-    return 'Flowdock';
+    if (robot.adapterName === 'flowdock') {
+      return 'Flowdock';
+    } else {
+      return 'HipChat';      
+    }
   }
 }
```

Comment:
Can we name this file just plain-text.js? No need to include transformer in the name of the file, this will match the slack and shell transformers.

Reply:
Okay, fixed!